### ISSUE TITLE
SMART-Seq v4 with stranded library

### ISSUE THREAD

**Issue by yeroslaviz** [alexdobin/STAR#1777]
Thanks for this great tool.
I have a simple question (I hope).

I'm doing a scRNA-Seq using the SMART-Seq library v4 from Takara labs. in the protocol it says, In the manual it is stated:

Read 1 generates sequences antisense to the original RNA.

What does it mean for me when setting the `--soloStrand` parameter? Do I need to set it to the default `forward`?

thanks

Assa

**Comment by alexdobin** [comment]
Hi Assa,

the ```--soloStrand``` parameter is determined by the 1st read, so it needs to be ```--soloStrand Reverse``` in this case.

Cheers
Alex

**Comment by yeroslaviz** [comment]
A followup question about that - 

Before mapping I ran `fastp` to trim/filter the reads.  The tool also has the option to keep the singeltons, where its mate didn't pass the filter threshold. 
There is an option to keep them in one file, which I also did (in retrospect maybe not the most wise decision). 

Is it possible to map these reads also onto the genome with the `--soloStrand unstranded` parameter (or with any of the other two)?
